app/views/scenarios/index.html.erb

```diff
@@ -21,7 +21,7 @@
         <% @scenarios.each do |scenario| %>
           <tr>
             <td>
-              <%= scenario_label(scenario, content_tag(:i, '', class: 'glyphicon glyphicon-font')) %>
+              <%= scenario_label(scenario, icon(scenario.icon)) %>
```

Comment:
Maybe we should pick a default icon that looks good and put it at the top?  It should also be used if `icon` on a Scenario is blank or nil.  Is there one you like?